frontend/libs/studio-components/src/components/StudioProperty/StudioPropertyButton/StudioPropertyButton.module.css

```diff
@@ -21,24 +21,36 @@
   white-space: nowrap;
 }
 
+.readOnly:hover {
+  cursor: auto !important;
+  background-color: transparent !important;
+}
+
 .property,
 .value {
   display: block;
   overflow: hidden;
   text-overflow: ellipsis;
 }
 
+.readOnlyWrapper,
 .editIconWrapper {
   flex: 1;
   text-align: right;
   display: none;
 }
 
 .propertyButton:hover .editIconWrapper,
-.propertyButton:focus .editIconWrapper {
+.readOnlyWrapper,
+.propertyButton:focus .editIconWrapper,
+.readOnlyWrapper {
```

Comment:
You are correct about the `.readOnlyWrapper` - this can be listed only once. But the `.editIconWrapper` is a descendant of the .propertyButton:- hover and focus classes, so I believe they must be present in both lines 🧐